src/api/function/UrlShortener.cs

```diff
@@ -79,15 +79,15 @@ ExecutionContext context
                 if (string.IsNullOrWhiteSpace(input.Url))
                 {
                     var badResponse = req.CreateResponse(HttpStatusCode.BadRequest);
-                    await badResponse.WriteAsJsonAsync(new { Message = "The url parameter can not be empty." });
+                    await badResponse.WriteAsJsonAsync(new { message = "The url parameter can not be empty." });
                     return badResponse;
                 }
 
                 // Validates if input.url is a valid aboslute url, aka is a complete refrence to the resource, ex: http(s)://google.com
                 if (!Uri.IsWellFormedUriString(input.Url, UriKind.Absolute))
                 {
                     var badResponse = req.CreateResponse(HttpStatusCode.BadRequest);
-                    await badResponse.WriteAsJsonAsync(new { Message = $"{input.Url} is not a valid absolute Url. The Url parameter must start with 'http://' or 'http://'." });
+                    await badResponse.WriteAsJsonAsync(new { message = $"{input.Url} is not a valid absolute Url. The Url parameter must start with 'http://' or 'http://'." });

```

Comment:
This looks like there was a typo in the original repo. It repeats `'http://'` twice instead of one of them being `'https://'`

Reply:
Good catch! I just pushed a change for it.